components/api_version/src/api_v1.rs

```diff
@@ -44,26 +46,20 @@ impl KvFormat for ApiV1 {
         match src_api {
             ApiVersion::V1 | ApiVersion::V1ttl => Ok(Key::from_encoded_slice(key)),
             ApiVersion::V2 => {
-                debug_assert_eq!(ApiV2::parse_key_mode(key), KeyMode::Raw);
-                let (mut user_key, _) = ApiV2::decode_raw_key(&Key::from_encoded_slice(key), true)?;
-                user_key.remove(0); // remove first byte `RAW_KEY_PREFIX`
-                Ok(Self::encode_raw_key_owned(user_key, None))
+                Err(EngineError::Other(box_err!("unsupported conversion")))
```

Comment:
```suggestion
                Err(box_err!("unsupported conversion"))
```
The same to other three similar.

Reply:
Maybe just panic ? The conversion from V2 to V1/V1TTL should be rejected by TiKV-BR, and when receive backup request. Other than the moment to convert key version.

Reply:
done, mark apiv2->apiv1 as unreachable